src/server/util/auth/auth.router.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router();
+const passport = require('passport');
+const authController = require('./auth.controller');
```

Comment:
Mixing `require` and `imports` again. Also please import before creating any objects if poss